src/components/grocery-view/grocery-section/GrocerySection.js

```diff
@@ -14,9 +12,10 @@ import styles from '../styles/grocery-section.module.css';
 
 // REFACTOR: Using fake setion titles
 const SECTIONS = ['Produce', 'Meat and Seafood', 'Bakery', 'Etc.'];
-const GROCERY_INFO = 'Click through product categories, view item grades, and add items to your cart straight from this page, or click an item to view more detailed information including details on its environmental impact.';
-
-// const scroll = (setStartIndex) => {};
+const GROCERY_INFO = `Click through product categories, view item grades,
+                      and add items to your cart straight from this page,
+                      or click an item to view more detailed information
+                      including details on its environmental impact.`;
```

Comment:
There are gonna be some merge conflicts with this and the DUMMY_DATA branch, they should be easy to resolve, I'll merge them when both are approved and figure out all the conflicts so don't worry about it.